### PR TITLE
Tweak test-integration.sh pipeline and add examples

### PR DESCRIPTION
**Pull request by pmorie**
@deads2k @pweil- 

Make it easier for the next guy to debug int tests.

### CODE REVIEW

**Comment by pmorie** [comment]
[test]

File: hack/test-integration.sh [hunk 1]
```diff
@@ -28,7 +28,9 @@ tags="${OS_TEST_TAGS:-integration no-docker}"
 
 export GOMAXPROCS="$(grep "processor" -c /proc/cpuinfo 2>/dev/null || sysctl -n hw.logicalcpu 2>/dev/null || 1)"
 TMPDIR=${TMPDIR:-/tmp}
-export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"}
+export BASETMPDIR=${BASETMPDIR:-"${TMPDIR}/openshift-integration"} 
+
+#"
```
Comment (deads2k) [inline hack/test-integration.sh]: My bash-fu must be weak.  Does this do anything?  It's just a weird comment line, right?

**Comment by deads2k** [comment]
Just formatting and examples?  I have no objection, just want to be sure I'm not missing anything.

**Comment by deads2k** [comment]
Also, I'm pretty sure bash best practice is to create the absolute longest lines possible to obfuscate what's actually happening for sysadmin job security :)

File: hack/test-integration.sh (hunk 1 above)
Comment (mfojtik) [inline hack/test-integration.sh]: i think it is just some left over from some refactoring

**Comment by mfojtik** [comment]
@deads2k from a dev point of view, it makes it harder to put comments to specific command :-) (and it also makes my vim cry)

File: hack/test-integration.sh (hunk 1 above)
Comment (pmorie) [inline hack/test-integration.sh]: My bad guys, the sublime colorization gets jacked up by line 31, that was my workaround to be able to edit the rest of the file without going blind.  I had it committed over here but not pushed.  You aren't crazy.

**Comment by pmorie** [comment]
> Also, I'm pretty sure bash best practice is to create the absolute longest lines possible to obfuscate what's actually happening for sysadmin job security :)

lol @deads2k

**Comment by pmorie** [comment]
Yeah, this is just formatting and examples.  It's not immediately obvious from reading the script that you can pass an argument to filter the tests.

**Comment by deads2k** [comment]
real dev's don't read help! :)

[merge]

**Pull request merged**